docs: Updated GSX Integration Docs to reflect current implementation (#999)
flybywiresim/docs: docs/fbw-a32nx/feature-guides/gsxintegration/fuel.md

## Usage
![Fuel Start](../../assets/gsxintegration/gsx-fuel-start.png)

> Much of the refuelling process between the A32NX and GSX is automated, however some input is required from the user to set the fuel and initiate refuelling.

Reviewer: ```suggestion
Much of the refuelling process between the A32NX and GSX is automated; However some input is required from the user to set the fuel and initiate refuelling.
```

1. Fueling
      - To use GSX as part of the refuelling process, 'Refuel Time' should be `Real` or `Fast`. Selecting `Instant` will add the fuel immediately, and will not involve GSX.

>       - Ensure you have synchronised your required fuel with your Simbrief flight plan, or manually adjust the slider to set how much fuel you require.

Reviewer: ```suggestion
      - Ensure you have synchronised your required fuel with your SimBrief flight plan, or manually adjust the slider to set how much fuel you require.
```

>       - Ensure you have syncronised your required fuel with your Simbrief flight plan, or manually adjusted the slider to set how much fuel you require.

Reviewer 2: ```suggestion
      - Ensure you have synchronised your required fuel with your Simbrief flight plan, or manually adjust the slider to set how much fuel you require.
```
